website/content/docs/configuration/telemetry.mdx

```diff
@@ -58,6 +58,15 @@ The following options are available on all telemetry configurations.
 - `publish_allocation_metrics` `(bool: false)` - Specifies if Nomad should
   publish runtime metrics of allocations.
 
+- `include_alloc_metadata_in_metrics` `(bool: false)` - This controls whether
+  allocation metadata should be part of the metric labels. Defaults to false
+  as enabling this exposes the risk of high cardinality labels. It is therefore
+  recommended to configure [allowed_metadata_keys_in_metrics](#allowed_metadata_keys_in_metrics).
+
+- `allowed_metadata_keys_in_metrics` `(list: [])` - This filters the metadata
+  keys to be included in the metric publishing. By default it will not filter
+  out any keys and thus include all metadata.
```

Comment:
```suggestion
- `include_alloc_metadata_in_metrics` `(bool: false)` - This controls whether
  allocation metadata is included in metric labels. Enabling this option may result in 
  high cardinality labels. You should also configure [allowed_metadata_keys_in_metrics](#allowed_metadata_keys_in_metrics).

- `allowed_metadata_keys_in_metrics` `(list: [])` - This filters the metadata
  keys to be included in the metric publishing. By default it will not filter
  out any keys and thus include all metadata.
```

@aimeeu want to weigh in on writing style here beyond my suggestion?

Reply:
Accepted your suggestion, leaving this comment open pending @aimeeu feedback